pkgs/by-name/_1/_1password-gui/sources.json

```diff
@@ -1,7 +1,7 @@
 {
   "stable": {
     "linux": {
-      "version": "8.11.12",
```

Comment:
downgrade ?

Reply:
botched branching. dfe99c3 was a prior attempt and is not part of this.
let me fix.